### DISCUSSION TITLE
Set sidebar menu to full width in mobile version if sidebar is collapsed

### DISCUSSION THREAD

**Discussion by thushi2007** [vikramlearning/blazorbootstrap#475] [Q&A]
I have the problem that the sidebar menu is still collapsed when I changed to mobile version. I want the sidebar resized to full width in mobile version. This problem also exists in the demo version on https://demos.blazorbootstrap.com/sidebar .

![image](https://github.com/vikramlearning/blazorbootstrap/assets/29477578/5a1848dd-e1cb-472c-b378-b7281716744a)

Have anyone already a solution for that?

**Comment by gvreddy04** [comment]
@thushi2007 Thank you for trying Blazor Bootstrap. Can you share a screen recording of the demo version which section you are trying to test in? This will help me to understand your problem instead of making assumptions.

**Reply by thushi2007** [reply]
@gvreddy04 Sure.  See the following gif:

![fk2_sidebar](https://github.com/vikramlearning/blazorbootstrap/assets/29477578/8a177e74-340f-458d-96e6-f71efbefcaa5)

I want the sidebar expanded, when I change to mobile version.
